extensions/sql-database-projects/src/test/project.test.ts

```diff
@@ -1002,18 +1002,19 @@ describe('Project: round trip updates', function (): void {
 		await testUpdateInRoundTrip(baselines.SSDTProjectBaselineWithBeforeBuildTarget);
 	});
 
-	// skipped until projectController does the prompting, test should be moved to projectController.tests.ts
```

Comment:
this test doesn't need to be moved to projectController.test.ts like the comment says?

Reply:
Nope; turned out that the flow for opening a project didn't really go through projectController (instead being data-workspaces calling into ISqlProject directly) so I switched the pattern to open projects from `new Project(path)` to `Project.OpenProject(path)`, 'cause it felt much less bad to have prompts in a static method rather than an instance method.

Reply:
There are still some warnings in the instance functions (warning when multiple pre/post deployment scripts) that I wouldn't be mad about moving out, but since those are prompt-less, it didn't feel as bad.